packages/fxa-settings/src/lib/types.ts

```diff
@@ -5,3 +5,61 @@
 export type LinkType = 'reset-password' | 'signin';
 
 export type LinkStatus = 'valid' | 'used' | 'expired' | 'damaged';
+
+// The following list was taken from the RP list in the admin panel.
+// It does not have as many entries because many entries (including Lockwise
+// and Mozilla Hubs, among others, had duplicate names,
+// and all we currently care about is the name string.
+// RPs have been alphabetized for easier reading.
+export enum RelyingParties {
+  addOns = 'Add-ons',
```

Comment:
Wahooooo!

Our naming convention for enums has been to start with a capital letter, I'm sorry to have to ask you to change these to `AddOns`, `Firefox`, `FirefoxMonitor` etc 🫣

I don't know that we need `firefoxAccounts = 'Firefox Accounts'`, weird that it's on the list since that's us? It might be good to run this list by Wil soon to see what's worth removing.

You are missing a close parenthesis in your comment btw!